engines/config-query-sparql/config/config-file.json

```diff
@@ -3,7 +3,7 @@
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/config-query-sparql/^4.0.0/components/context.jsonld"
```

Comment:
This `config-file.json` (and the next 2 files) can not be changed to ensure backwards-compat. Instead, a copy needs to be made with the `-v4-2-0.json` suffix, and that one needs to be referenced in the engines.

Reply:
jus to confirm:
`engines/config-query-sparql/config/config-file-v4-2-0.json`
`engines/config-query-sparql/config/config-rdfjs-lite-v4-2-0.json`
`engines/config-query-sparql/config/config-rdfjs-v4-2-0.json`

and how do we fix the dependency error? I thought to switch the `actor-query-operation-distinct-identity` for the `actor-query-operation-distinct-hash` but this seems not to be correct.

Reply:
Should be correct, yes.

>  I thought to switch the actor-query-operation-distinct-identity for the actor-query-operation-distinct-hash but this seems not to be correct.

That is needed indeed.

> and how do we fix the dependency error?

The reported dependencies will have to be added. (some other deps are reported due to other actors being necessary for `v4-1-0`, but those weren't added in the last update yet, but will have to be included now)

Reply:
I updated the package.json to fix the depcheck error. so I removed the distinct-identity from sparql-rdf and sparql-rdf-lite